packages/tupaia-web/src/components/DateRangePicker.tsx

```diff
@@ -77,9 +76,9 @@ const DialogPaperComponent = styled.div`
 `;
 
 interface DateRangePickerProps {
-  startDate?: Moment;
-  endDate?: Moment;
-  granularity?: ValueOf<typeof GRANULARITIES>;
+  startDate?: Moment | string;
+  endDate?: Moment | string;
+  granularity?: `${VizPeriodGranularity}`;
```

Comment:
Using a type like `${.....}` makes the dev-x a little nicer because when you hover over the value it shows all the string values it could be